modules/doc/content/application_usage/ondemand.md

```diff
@@ -0,0 +1,72 @@
+# Open OnDemand
+
+[Open OnDemand](https://openondemand.org) is a web based HPC access portal. OnDemand allows users to access HPC files, open shells, access Linux desktops, use other interactive apps, and build job submissions to submit to a cluster, all from the web browser.  From the OnDemand web portal, users who have been given permission, are able to use level 1 code access.
```

Comment:
Is the purpose of this page to be for OnDemand in general, or hpcondemand?

If it's the latter, I suggest combining this paragraph and the one below.

Reply:
OnDemand in general. The apps can be accessed either internally or externally (Ondemand or HPCOndemand).